apps/meteor/client/views/marketplace/AppDetailsPage/tabs/AppDetails/AppDetails.tsx

```diff
@@ -9,6 +10,18 @@ import ScreenshotCarouselAnchor from '../../../components/ScreenshotCarouselAnch
 import type { AppInfo } from '../../../definitions/AppInfo';
 import AppDetailsAPIs from './AppDetailsAPIs';
 
+const normalizeUrl = (url: string): string => {
+	const parsedUrl = parse(url);
+
+	if (parsedUrl[0].type === 'PARAGRAPH') {
+		if (parsedUrl[0].value[0].type === 'LINK') {
+			return parsedUrl[0].value[0].value.src.value;
+		}
+	}
+
+	return url;
```

Comment:
Prefer early returns.

```suggestion
	const parsedUrl = parse(url);

	if (parsedUrl[0].type !== 'PARAGRAPH') {
		return url;
	}
	
	if (parsedUrl[0].value[0].type !== 'LINK') {
		return url;
	}
	
	return parsedUrl[0].value[0].value.src.value;
```